content/docs/connectors.md

```diff
@@ -0,0 +1,634 @@
+Also it is quite important for you to identify which are the parameters used, in particular the mandatory ones and which are needed for producer or for consumer purpose and their type: typically they are text representing a configuration, but in some circumstances they may represent more complex java objects such as a Datasource or a connection bean of some sort.
+
+I would even suggest to make some experiment and test directly on the upstream component in order to familiarize with the component itself. Finally, your goal is to integrate on Syndesis, but this will end up in a Camel route deployed (probably to a container), so, you better know what to expect there. There are [endless example you can run](https://github.com/apache/camel/tree/master/examples), so you will be easily find the one for your component too.
+
+At this stage we should have enough clarity on what our component does, which is the expected URI structure and the expected configuration settings.
+###### Changes in existing camel components
+If you are going to make any changes to the existing camel components, in order to be able to use them properly make sure to bring the correct version and to rebuild the `camel-catalog` dependency. This one is used at runtime by Syndesis to read configuration information such as the expected parameters, so, any change must be reflected by refreshing this dependency as well.
+### Maven project structure
+As there is not yet an archetype available, we need to make some manual `pom.xml` configuration in order to prepare our project. In order to maintain the original structure of the project you will have to create a connector folder under the `app/connector` directory. In order to maintain the convention, call it `connector-nameOfComponent` (ie, `connector-mongodb`).
+
+Next step is to create the minimum viable pom in order to be able to wrap all the needed dependencies for our connector.
+###### Connector description
+Almost all related to the project is managed by the connector parent pom. So, you just have to provide the name of this new dependency:
+
+    <?xml version="1.0" encoding="UTF-8"?>
+    <!-- Copyright (C) 2016 Red Hat, Inc. Licensed under the Apache ...-->
+    <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+        <parent>
+            <groupId>io.syndesis.connector</groupId>
+            <artifactId>connector-parent</artifactId>
+            <version>1.7-SNAPSHOT</version>
+            <relativePath>../pom.xml</relativePath>
+        </parent>
+    
+        <modelVersion>4.0.0</modelVersion>
+        <artifactId>connector-mongodb</artifactId>
+        <name>Connector :: MongoDB</name>
+        <packaging>jar</packaging>
+
+You will have to change the version and name of your connector accordingly. 
```

Comment:
Should we mention where can we find the proper version or would that be too dummy?